generator.yaml

```diff
@@ -526,6 +528,40 @@ resources:
         from:
           operation: DescribeFeatureGroup
           path: FeatureGroupStatus
+  ModelPackageGroup:
+      exceptions:
+        errors:
+            404:
+              code: ValidationException
+              message_suffix: does not exist.
+        terminal_codes:
+          - ResourceLimitExceeded
+          - ResourceNotFound
+          - ResourceInUse
+          - OptInRequired
+          - InvalidParameterCombination
+          - InvalidParameterValue
+          - MissingParameter
+          - MissingAction
+          - InvalidQueryParameter
+          - MalformedQueryString
+          - InvalidAction
+          - UnrecognizedClientException
+      fields:
+        ModelPackageGroupStatus:
+          is_read_only: true
+          print:
+            name: STATUS
+          from:
+            operation: DescribeModelPackageGroup
+            path: ModelPackageGroupStatus      
+      hooks:
+        sdk_create_post_set_output:
+          code: rm.customSetOutput(&resource{ko})
```

Comment:
unfortunately, this is not useful for sagemaker because create never returns status which this method is looking at

so am curious, does the `Status.ModelPackageGroupStatus` reflects completed with current code? if not we need to inject status manually

sorry, didn't realize this in previous revisions

Reply:
yes it reflects as completed since all other states aren't currently used by sagemaker they are just there if they implement them in the future